File: C Sharp/judgingmoose.cs
using System;

public class judgingmoose {
    public static void Main (string[] args) {
        string line = Console.ReadLine ();
        int space = line.IndexOf (' ');
        int left = int.Parse (line.Substring (0, space));
        int right = int.Parse (line.Substring (space + 1));
        Console.WriteLine (left + right == 0 ? "Not a moose" : left == right ? "Even " + (right << 1) : "Odd " + (right > left ? right << 1 : left << 1));
        }
    }

File: Java/judgingmoose.java

// https://open.kattis.com/problems/judgingmoose
import java.util.Scanner;

public class judgingmoose {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int left = sc.nextInt();
        int right = sc.nextInt();

        System.out.println(left == 0 && right == 0 ? "Not a moose"
                : left == right ? "Even " + (left + right) : left > right ? "Odd " + (left * 2) : "Odd " + (right * 2));
        sc.close();
    }
}